client/src/components/Timedown.vue

```diff
@@ -20,22 +20,33 @@ function decrementSeconds() {
   if (secondsLeft.value > 0) secondsLeft.value--;
 }
 
+const msToTimeLeft = (ms: number) => {
+  let min = Math.floor(ms / 60000);
+  let sec = Number.parseInt(((ms % 60000) / 1000).toFixed(0));
+  return min + ":" + (sec < 10 ? "0" : "") + sec;
+};
+
 const secondsLeft = ref(props.currentSeconds);
 const interval = ref(null);
 const miliseconds = computed(() => secondsLeft.value * 1000);
-const displayValue = computed(() =>
-  humanizeDuration(miliseconds.value, { language: i18n.global.locale })
```

Comment:
Did this not work as required?

Reply:
It works as it is supposed but it doesn't have any configuration for what we need. The module is thought to give you a detailed description of the time (1 hour, 20 minutes and 5 seconds + localizations) but according to the prototypes we just need a `mm:ss`, without any text or translations.

https://www.npmjs.com/package/humanize-duration